components/alert/test/alert-toast-helper.js

```diff
@@ -1,26 +1,21 @@
-
-module.exports = {
-
-	async getRect(page, selector) {
-		const rect = await page.$eval(selector, (toast) => {
-			const elem = toast.shadowRoot.querySelector('.d2l-alert-toast-container');
-			return {
-				x: elem.offe,
-				y: elem.offsetTop,
-				width: elem.offsetwidth,
-				height: elem.offsetHeight
-			};
-		});
+export async function getRect(page, selector) {
+	const rect = await page.$eval(selector, (toast) => {
+		const elem = toast.shadowRoot.querySelector('.d2l-alert-toast-container');
 		return {
-			x: 0,
-			y: rect.y - 10,
-			width: page.viewport().width,
-			height: page.viewport().height - rect.y + 10
+			x: elem.offe,
```

Comment:
This `offe` property looks a bit weird.  Is/was it `undefined`, and we were returning `0` before?  The `offsetwidth` one looks a bit strange too, since the native API is `offsetWidth`.

Reply:
I agree this looks wrong, but this didn't actually change, I just removed the indent (the diff looks weird but if you look at side-by-side it'll be clearer).

Reply:
The code appears to have been always like this since Allan introduced it. The good news: it's not actually used, so I've updated it a bit to only expose the used stuff.